x/fungible/keeper/zrc20_lock_token.go

```diff
@@ -0,0 +1,82 @@
+package keeper
+
+import (
+	"fmt"
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+
+	fungibletypes "github.com/zeta-chain/node/x/fungible/types"
+)
+
+const transferFrom = "transferFrom"
+
+var zeroAddress common.Address = common.HexToAddress("0x0000000000000000000000000000000000000000")
```

Comment:
We already have it defined as a constant in a package

Reply:
Using the high level function provided by address.go here: 445b5c3287eeec113488bbfec6f8bcd26ee83413